ch09/ex-event-bubbling.md

```diff
@@ -153,3 +153,27 @@ Both should be able to work on the same click.
     >>> js = this_browser.active_tab.js
     >>> js.run("document.querySelectorAll('input')[0].getAttribute('value')")
     'sugar'
+
+
+
+
+
+
+Test: Form submit still works after event bubbling changes
+
+    >>> script = """
+    ... document.querySelectorAll('input')[0].addEventListener('click',
+    ...   function(e) {
+    ...     console.log('input saw a click');
+    ...   });
+    ... """
+    >>> js_url = wbemocks.socket.serve(script)
+    >>> body = "<script src=" + str(js_url) + "></script>"
+    >>> body += "<form action='http://example.com/submit' method='post'><input type='submit' value='Submit'></form>"
+    >>> html_url = wbemocks.socket.serve(body)
+
+    >>> this_browser = browser.Browser()
+    >>> this_browser.new_tab(browser.URL(html_url))
+    >>> this_browser.handle_click(wbemocks.ClickEvent(20, 24 + this_browser.chrome.bottom))
+    input saw a click
```

Comment:
I don't think this is enough: it shows we handled a click, but the form is not submitted (because it is an input). Proposal:
- Change the `<input>` to `<button>` so that clicking it submits the form; similarly change the event handler
- Add one more line to the test that checks that the active tab's URL is now `http://example.com/submit`, as it should be after submitting the form.